### PR TITLE
implement waiting for databases to re-occur after backup restore

### PR DESCRIPTION
**Pull request by dothebart**
the attempt to do this inside of checkdata wasn't as good:
https://github.com/arangodb/rta-makedata/pull/14/commits/ed477374dcbbb4eeecf9aa6d8b6cb4926513054a
this way is the way to go.

### CODE REVIEW

File: release_tester/arangodb/sh.py [hunk 1]
```diff
@@ -244,6 +244,31 @@ def hotbackup_check_for_nonbackup_data(self):
 
         return res
 
+    @step
+    def hotbackup_wait_for_ready_after_restore(self):
+        """check whether the data is in there or not."""
+        logging.info("running version check")
+        #  || db.this_collection_will_not_be_backed_up._length() != 0
```
Comment (alexbakharew) [inline release_tester/arangodb/sh.py:251]: Please remove debugging leftovers comments

File: release_tester/arangodb/sh.py [hunk 2]
```diff
@@ -244,6 +244,31 @@ def hotbackup_check_for_nonbackup_data(self):
...
+        let timeout = 15;
+        while (db._databases().length == 0)  {
+          if (timeout == 0) {		
+            throw new Error("Databases array is still empty after 15s!");		
+          }		
+          require("internal").sleep(1);		
+          timeout -= 1;
+        }
+        """
+        logging.debug("script to be executed: " + str(js_script_string))
```
Comment (alexbakharew) [inline release_tester/arangodb/sh.py:266]: Why we need to put entire script in output? Can we just print message that we are waiting for database's array?

File: release_tester/arangodb/sh.py [hunk 3]
```diff
@@ -244,6 +244,31 @@ def hotbackup_check_for_nonbackup_data(self):
 
         return res
 
+    @step
+    def hotbackup_wait_for_ready_after_restore(self):
+        """check whether the data is in there or not."""
```
Comment (alexbakharew) [inline release_tester/arangodb/sh.py:249]:
```suggestion
        """check that array with databases is not empty."""
```

File: release_tester/arangodb/sh.py [hunk 4]
```diff
@@ -244,6 +244,31 @@ def hotbackup_check_for_nonbackup_data(self):
...
+        logging.info("running version check")
```
Comment (alexbakharew) [inline release_tester/arangodb/sh.py:250]: Wrong logging message?

**Review by alexbakharew — CHANGES_REQUESTED** [review]
Left some comments

File: release_tester/arangodb/sh.py (hunk 2 above)
Comment (dothebart) [reply release_tester/arangodb/sh.py:266]: I had the issue of too much magic happening a while back, and hence printed it everywhere.

**Pull request merged**